### ISSUE TITLE
[BUG] Menu bar accessibility. No programmatically determinable announcements when moving between menu items in menu bar

### ISSUE THREAD

**Issue by spudthebud** [wet-boew/wet-boew#9093]
## Required information

**Describe the bug**
An accessibility problem is present in the menu systems.  When using JAWS and pressing left or right in a menu bar, there is a clear visual change indicating the menu that has focus or an open menu. However, there is no aural notice to tell the JAWS users that they are on a new menu or sub-menu. 

If you found another bug report related to this issue on GitHub, please mention its number e.g. #7890

**To Reproduce**
1. Using JAWS 2021 on Chrome, go to https://www.justice.gc.ca/eng/
2. Tab until you get to the first menu item in the menu bar, Family Law.  You should be moved to "Changes in Family Law" 
3. Press the right key to move to the next menu in the menu bar, Criminal Justice.
4. Notice that there is a visual event as the menu opens. However, there is no aural event announcing the change.

**Expected behavior**
When pressing left or right to open neighboring menus in the menu bar, there should be some sort of aural announcement, such as <menu name, expanded>

**Who shall do the work?**
I am asking for Principal publisher to please do the work.

**Version of WET-BOEW/GCWeb you are using**
Whatever version is on https://www.justice.gc.ca/eng/ and other government sites currently. 

**Desktop/Smartphone (please complete the following information)**
 - OS or Device: Windows 10
 - Browser: Chrome 90; Edge 90
 - JAWS 2021

**Screenshots**
See video recording 

https://user-images.githubusercontent.com/8799391/117088243-ddb2b500-ad1f-11eb-80c5-9d0e422b3bd2.mp4

**Comment by thekodester** [comment]
This website (https://www.justice.gc.ca/eng/) is using v4.0.27 - 2017-12-14. Be up to that department to update their site. Though I'm sure @duboisp can send your feedback to them.

Are you able to reproduce your issue on a more modern version of wet such as:
https://wet-boew.github.io/wet-boew/index-en.html
https://www.canada.ca/en.html

###### I am just a volunteer on this GitHub project.

**Comment by spudthebud** [comment]
No, there are still similar problems on the github site example. Maybe even more problems.  Can some describe how WET is tested with JAWS before releases?

**Comment by thekodester** [comment]
I would not know the testing process as volunteer. But what issues are you having with https://wet-boew.github.io/wet-boew/index-en.html ?

**Comment by spudthebud** [comment]
pretty much the same major one in the video. Is there a way to notify folks about the accessibility aspect of this?

**Comment by thekodester** [comment]
Your thoughts @shawnthompson ?

**Comment by shawnthompson** [comment]
As far as I know WET-BOEW isn't tested with AT devices before releases. It's an open source project and they rely on people like you, @spudthebud, to report the bugs and have someone fix them. 

I will bring this up with my team to see what the issue is. 

Stay tuned

**Comment by shawnthompson** [comment]
By The Way it's not working in VoiceOver either. I'd like to redo the menu. It's using the best practices for application menu but really it's just a navigation menu. Huge discussion about this in the accessibility community at large but no official.

Here's a great article by @aardrian

[Link + Disclosure Widget Navigation](https://adrianroselli.com/2019/06/link-disclosure-widget-navigation.html)

**Comment by spudthebud** [comment]
@shawnthompson  I understand this is an open source project, but it's also a project of the government, so subject to the Charter.  This open source format is a helpful way to receive feedback but it can't be relied on to ensure accessibility features are built in. Surely AT testing is done in advance, or the government hires AT experts to do this important work.

**Comment by shawnthompson** [comment]
I completely agree with you @spudthebud. I'll look into this problem and hopefully have a fix soon. Then the Justice site will have to update to the new version of WET.
